dependencies.py

```diff
@@ -0,0 +1,32 @@
+#!usr/bin/env/python
+
+import os
+import time
+
+Y = set(['yes', 'y', 'YES', 'Y'])
+N = set(['no', 'n', 'NO', 'N'])
+
+os.system("apt install figlet")
+os.system("figlet INSTALLATION")
+os.system("clear")
```

Comment:
Why generate a banner if right after you are going to remove it?
For that it is better not to put any banner